distribution/linux/start.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env sh
+# We recommend using Pine-jinx if you want the game to be optimised and installed for your system. This is mostly for PR builds and people who want a portable install.
+# It is also recommended to run start.sh in the terminal, although this isn't required in order for it to operate
+SCRIPT_DIR=$(dirname $0)
```

Comment:
```suggestion
SCRIPT_DIR=$(dirname $(realpath $0))

```
Just in case someone starts to mess with symlinks.